index.html

```diff
@@ -73,6 +70,253 @@ <h1>I’m Sebastian. Glad to see you!</h1>
+                </ul>
+              </div>
+
+              <button class="see-proj-btn-itself" type="button">
+                See Project
+              </button>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section class="about-myself headline">
+      <div class="intro-content intro">
+        <h1 class="title-about-myself">About Myself</h1>
+        <p>
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+
+        <div class="connect-div">
+          <a href="#" class="connect-a">
+            <h3>LET'S CONNECT</h3>
+          </a>
+        </div>
+
+        <div class="social-icons about-ms">
+          <ul>
+            <li>
+              <a href="#"><i class="fa fa-instagram"></i></a>
+            </li>
+            <li>
+              <a href="#"><i class="fa fa-twitter"></i></a>
+            </li>
+            <li>
+              <a href="#"> <i class="fa fa-github"></i> </a>
+            </li>
+            <li>
+              <a href="#"><i class="fa fa-linkedin"></i></a>
+            </li>
+          </ul>
+        </div>
+
+        <button class="see-proj-btn-itself resume-btn" type="button">
+          Get my resume
+        </button>
+      </div>
+
+      <div class="about-second-section intro">
+        <ul class="accordion-menu">
+          <li>
+            <div class="dropdownlink">
+              <h3 class="about-titles" id="Language-title">Languages</h3>
+              <i class="fa fa-chevron-down arrows" aria-hidden="true"></i>
+            </div>
+            <ul class="submenuItems">
+              <li>
+                <img
+                  src="./styling-features-pictures/JS-CIRCLE.png"
+                  alt=""
+                  class="circle-menu-pic"
+                />
+                <a href="#">JavaScript</a>
+              </li>
+              <li>
+                <img
+                  src="./styling-features-pictures/HTML-CIRCLE.png"
+                  alt=""
+                  class="circle-menu-pic"
+                />
+                <a href="#">HTML</a>
+              </li>
+              <li class="no-margin-bottom">
+                <img
+                  src="./styling-features-pictures/CSS-CIRCLE.png"
+                  alt=""
+                  class="circle-menu-pic"
+                />
+                <a href="#">CSS </a>
+              </li>
+            </ul>
+          </li>
+
+          <li>
+            <div class="dropdownlink bottom-separator">
+              <h3 class="about-titles" id="fr-title">Frameworks</h3>
+              <i class="fa fa-chevron-down arrows" aria-hidden="true"></i>
+            </div>
+          </li>
+
+          <li>
+            <div class="dropdownlink bottom-separator">
```

Comment:
- [OPTIONAL] kindly make sure to use the horizontal flesh when the link is not clickable as we can see in the [Figma](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/images/m1_setup/about-myself-mobile.png) template

Reply:
Ok, already done it!